src/tests/unit/electron/adapters/electron-storage-adapter.test.ts

```diff
@@ -13,75 +13,50 @@ describe('ElectronStorageAdapter', () => {
         testKey1: 'test-value-1',
         testKey2: 'test-value-2',
     };
-    let electronStorageAdapter: ElectronStorageAdapter;
+
     let indexedDBInstanceMock: IMock<IndexedDBAPI>;
     let loggerMock: IMock<Logger>;
 
+    let testSubject: ElectronStorageAdapter;
+
     beforeEach(() => {
         indexedDBInstanceMock = Mock.ofType<IndexedDBAPI>();
         loggerMock = Mock.ofType<Logger>();
-        electronStorageAdapter = new ElectronStorageAdapter(indexedDBInstanceMock.object, loggerMock.object);
+        testSubject = new ElectronStorageAdapter(indexedDBInstanceMock.object, loggerMock.object);
     });
 
-    describe('setting user data with electron storage adapter works', () => {
-        it('sets user data with input items', async () => {
+    describe('setUserDataP', () => {
+        it('succeed', async () => {
             indexedDBInstanceMock
-                .setup(async indexedDB => await indexedDB.setItem(IndexedDBDataKeys.installation, It.isValue(expectedData)))
+                .setup(indexedDB => indexedDB.setItem(IndexedDBDataKeys.installation, It.isValue(expectedData)))
                 .returns(() => Promise.resolve(true))
                 .verifiable(Times.once());
 
-            loggerMock.setup(logger => logger.error(It.isAny())).verifiable(Times.never());
+            await testSubject.setUserData(expectedData);
 
-            electronStorageAdapter.setUserData(expectedData);
-            await tick();
             indexedDBInstanceMock.verifyAll();
-            loggerMock.verifyAll();
+            loggerMock.verify(logger => logger.error(It.isAny()), Times.never());
         });
 
-        it('sets user data and then fires the callback properly', async () => {
-            const mockCallback = jest.fn(() => {
-                // do nothing
-            });
-            indexedDBInstanceMock
-                .setup(async indexedDB => await indexedDB.setItem(IndexedDBDataKeys.installation, It.isValue(expectedData)))
-                .returns(() => Promise.resolve(true))
-                .verifiable(Times.once());
-
-            loggerMock.setup(logger => logger.error(It.isAny())).verifiable(Times.never());
-
-            electronStorageAdapter.setUserData(expectedData, mockCallback);
-
-            await tick();
-            expect(mockCallback).toHaveBeenCalled();
-            indexedDBInstanceMock.verifyAll();
-        });
+        it('throws', async () => {
```

Comment:
`it throws` doesn't really tell me enough to understand what requirement this is verifying. `it propagates exceptions from indexedDB as-is`? (similar for other test names)

Reply:
Great suggestion for the name of the test! Will update